scripts/ci/setup-dev-sso.sh

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env bash
+
+user_help () {
+    echo "Deploy in cluster keycloak and configure registration service to use it."
+    echo "options:"
+    echo "-sn, --sso-ns  Builds and pushes the operator to quay"
+    echo "-h,  --help              To show this help text"
+    echo ""
+    exit 0
+}
+
+read_arguments() {
+    if [[ $# -lt 2 ]]
+    then
+        echo "There are missing parameters"
+        user_help
+    fi
+
+    while test $# -gt 0; do
+           case "$1" in
+                -h|--help)
+                    user_help
+                    ;;
+                -sn|--sso-ns)
+                    shift
+                    DEV_SSO_NS=$1
+                    shift
+                    ;;
+                *)
+                   echo "$1 is not a recognized flag!" >> /dev/stderr
+                   user_help
+                   exit -1
+                   ;;
+          esac
+    done
+}
+check_commands()
+{
```

Comment:
minor, cosmetic: it may be missing an empty line

```suggestion
}

check_commands()
{
```